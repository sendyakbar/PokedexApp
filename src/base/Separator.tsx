import React from 'react';
import {StyleSheet, View, ViewStyle} from 'react-native';

export default function Separator(): JSX.Element {
  return <View style={styles.container} />;
}

type StyleType = {
  container: ViewStyle;
};

const styles = StyleSheet.create<StyleType>({
  container: {
    width: '100%',
    height: 0.5,
    backgroundColor: '#dddddd',
    borderRadius: 100,
  },
});
