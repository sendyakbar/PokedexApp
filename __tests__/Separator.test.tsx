import 'react-native';
import React from 'react';
import {it} from '@jest/globals';
import renderer from 'react-test-renderer';

import Separator from '../src/base/Separator';

it('renders correctly', () => {
  const tree = renderer.create(<Separator />).toJSON();
  expect(tree).toMatchSnapshot();
});
