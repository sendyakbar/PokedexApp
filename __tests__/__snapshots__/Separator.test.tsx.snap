// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#dddddd",
      "borderRadius": 100,
      "height": 0.5,
      "width": "100%",
    }
  }
/>
`;
